### PR TITLE
Swagger updates for scan job

### LINKED ISSUE #1096: Review Swagger Docs: Scan Job section
## Specify type:
- Enhancement

### Priority:
- Medium
___

## Description:
Review the scan job APIs for correct definition and descriptions. Improve the associated example information
___

## Acceptance Criteria:

- [ ] Verify that swagger docs are awesome for the Scan Job API

### PR DESCRIPTION
**Pull request by kholdaway**
Closes #1096

### CODE REVIEW

File: docs/swagger.yml [hunk 1]
```diff
@@ -638,18 +636,20 @@ paths:
...
-      description: "List all scans."
+      description: "List all scans jobs for a specific scan"
       operationId: "listScanJobs"
       produces:
         - "application/json"
       parameters:
         - name: "scan_id"
           in: "path"
-          description: "Scan used to create job."
+          description: "Scan id use in search"
```
Comment (abaiken) [inline docs/swagger.yml]: `ID of scan used in search` to be consistent with description on line 733.

File: docs/swagger.yml [hunk 2]
```diff
@@ -759,21 +763,23 @@ paths:
...
-      description: "Get the inspection results of a scan."
+      description: "Get the inspection results of a scan"
       operationId: "getScanInspectionResults"
       produces:
       - "application/json"
       parameters:
       - name: "scan_job_id"
         in: "path"
-        description: "ID of scan job to retrieve inspection results for"
+        description: "ID of scan job to use in search"
```
Comment (abaiken) [inline docs/swagger.yml]: Same comment as above. We should make the descriptions consistent --> either `ID of scan job used in search` or `ID of scan job to use in search`.

File: docs/swagger.yml [hunk 3]
```diff
@@ -831,14 +839,16 @@ paths:
...
+          description: "Not authorized"
         404:
           description: "Scan job not found"
   /jobs/{scan_job_id}/cancel/:
     put:
       tags:
         - "Scan Job"
       summary: "Cancels an existing scan job"
-      description: "Cancels a scan if its has not finished, fails otherwise."
+      description: "Cancels a scan if its has not finished, fails otherwise"
```
Comment (abaiken) [inline docs/swagger.yml]: `its has not` --> `it has not`

File: docs/swagger.yml [hunk 4]
```diff
@@ -856,14 +866,16 @@ paths:
...
+          description: "Not authorized"
         404:
           description: "Scan job not found"
   /jobs/{scan_job_id}/restart/:
     put:
       tags:
         - "Scan Job"
       summary: "Restarts an existing scan job"
-      description: "Restarts a scan if its has been paused, fails otherwise."
+      description: "Restarts a scan if its has been paused, fails otherwise"
```
Comment (abaiken) [inline docs/swagger.yml]: Same as above: `its has not` --> `it has not`

File: docs/swagger.yml [hunk 5]
```diff
@@ -1484,19 +1511,23 @@ definitions:
...
             type: "integer"
             format: "int64"
-            description: "The total number of systems that failed to scan."
+            description: "The total number of systems that failed to scan"
+            example: 10
           report_id:
             type: "integer"
             format: "int64"
-            description: "The identifier of the sent fact collection."
+            description: "The identifier of the sent fact collection"
```
Comment (abaiken) [inline docs/swagger.yml]: Just a suggestion but maybe change this to `The fact collection identifier` or `Fact collection identifier` to be consistent with line 2017.

File: docs/swagger.yml [hunk 6]
```diff
@@ -806,14 +812,16 @@ paths:
...
+          description: "Not authorized"
         404:
           description: "Scan job inspection results not found"
   /jobs/{scan_job_id}/pause/:
     put:
       tags:
         - "Scan Job"
       summary: "Pauses an existing scan job"
-      description: "Pauses a scan if its currently running, fails otherwise."
+      description: "Pauses a scan if its currently running, fails otherwise"
```
Comment (abaiken) [inline docs/swagger.yml]: `its` -> `it is`

**Comment by kholdaway** [comment]
Great review @abaiken

**Comment by kholdaway** [comment]
Addressed all feedback from review

**Pull request merged**